### ISSUE TITLE
Only first unreachable statement flagged in 'case'

### ISSUE THREAD

**Issue by ikonst** [microsoft/TypeScript#55562]
### 🔎 Search Terms

In `case`, only the first unreachable statement is flagged. This is inconsistent e.g. with function scope.

### 🕗 Version & Regression Information

Reproduces on 5.2.2 and nightly.

### ⏯ Playground Link

See [Playground](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMAFASkQbwFCMQJwFMoR8wBuXRCBAZzgBtCA6BuAcxQCIBDLtSnhph6TVh258B2AL7ZsoSLASJOGHHloB3GFAgALRJP5YqQnrUKJeXAFxm8BYqQoOhdRizacb0x4+FRLwlfQUQ5OWxCAA8ABzh8KCwZSiA)

### 💻 Code

```typescript
function g() {
  switch ("a") {
    case "a":
      return;
      console.log("a"); // grayed out
      console.log("a"); // not grayed out
  }
}
```

### 🙁 Actual behavior

First statement grayed out.

### 🙂 Expected behavior

Both statements grayed out.

**Comment by piotrzarycki** [comment]
can i take this bug :) ?

**Comment by Andarist** [comment]
https://github.com/microsoft/TypeScript/blob/08e0eb74e6eb1acac26221363c1b1fffe5102d47/CONTRIBUTING.md#issue-claiming

Please let me know if you intend to work on this or if you decide not to in the end cause I already have a potential fix for this locally 😅 I don't have to push it out though - if you want to take a stab at it, please do! It's always nice to see more people contributing :)

**Comment by fatcerberus** [comment]
That was entertaining; cookie-licking doesn’t usually go in both directions 😄
